app/bundles/EmailBundle/Form/Type/EmailSendType.php

```diff
@@ -186,6 +195,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 );
             }
         }
+        if (!empty($options['with_immediately'])) {
+            if ($this->coreParametersHelper->getParameter('mailer_spool_type') == 'file') {
+                $default = (isset($options['data']['immediately'])) ? $options['data']['immediately'] : false;
+                $builder->add(
+                'immediately',
+                YesNoButtonGroupType::class,
+                [
+                    'label' => 'mautic.form.action.sendemail.immediately',
+                    'data'  => $default,
+                    'attr'  => [
+                        'tooltip'    => 'mautic.form.action.sendemail.immediately.desc',
+                    ],
+                ]
+            );
```

Comment:
There is something fishy about the indentation here. Shouldn't this line (and some above) move 4 spaces right? How come CS Fixer is good with that?